data.sql

```diff
@@ -42,4 +42,39 @@ UPDATE animals set owner_id = 3 WHERE name in ('Devimon','Plantmon');
 UPDATE animals set owner_id = 4 WHERE name in ('Charmander','Squirtle','Blossom');
 
 UPDATE animals set owner_id = 5 WHERE name in ('Angemon','Boarmon');
-       
\ No newline at end of file
+       
+--        Day 4
+
+INSERT INTO vets(id,name, age, date_of_graduation)
+	VALUES 	(1,'William Tatcher', 45, '2000-4-23'),
+			(2,'Maisy Smith', 26, '2019-2-17'),
+			(3,'Stephanie Mendez', 64, '1981-5-4'),
+			(4,'Jack Harkness', 38, '2008-1-8');
+
+INSERT INTO specializations (id,vet_id, species_id)
+	VALUES 	(1,1,1),
+			(2,3,1),
+			(3,3,2),
+			(4,4,2);
+
+INSERT INTO visits(id,animal_id, vet_id, date_of_visit) 
+VALUES	(1,1,1,'2020-05-24'),
+			(2,1,3,'2020-07-22'),
+			(3,2,4,'2021-02-02'),
+			(4,3,2,'2020-01-05'),
+			(5,3,2,'2020-03-08'),
+			(6,3,2,'2020-05-14'),
+			(7,4,3, '2021-05-04'),
+			(8,5,4, '2021-02-24'),
+			(9,6,2, '2019-12-21'),
+			(10,6,1, '2020-08-10'),
+			(11,6,2, '2021-04-07'),
+			(12,7,3, '2019-09-29'),
+			(13,8,4, '2020-08-03'),
+			(14,8,4, '2020-11-04'),
+			(15,9,2, '2019-01-24'),
+			(16,9,2,'2019-05-15'),
+			(17,9,2, '2020-02-27'),
+			(18,9,2,'2020-08-03'),
+			(19,10,3,'2020-05-24'),
+			(20,10,1, '2021-01-11');
```

Comment:
- [ ] The same thing with the visits table, there is no need to add the id manually.